test/gaugeTest.js

```diff
@@ -43,19 +42,21 @@ describe('gauge', () => {
 			});
 
 			it('should start a timer and set a gauge to elapsed in seconds', () => {
-				const clock = lolex.install();
+				jest.useFakeTimers('modern');
+				jest.setSystemTime(0);
 				const doneFn = instance.startTimer();
-				clock.tick(500);
+				jest.advanceTimersByTime(500);
 				doneFn();
 				expectValue(0.5);
-				clock.uninstall();
+				jest.useRealTimers();
 			});
 
 			it('should set to current time', () => {
-				const clock = lolex.install();
+				jest.useFakeTimers('modern');
+				jest.setSystemTime(0);
 				instance.setToCurrentTime();
 				expectValue(Date.now());
```

Comment:
example of broken test. `Date.now() === 0` so value is also `0`. Setting it to an actual time gives test errors like this:

![image](https://user-images.githubusercontent.com/1404810/81565615-ea652d00-9399-11ea-9bdb-3c9f5a9bb24a.png)